### PR TITLE
Add 20 records in seed.rb to generate 20 items when call rake db:setup

### PR DESCRIPTION
**Pull request by HaominZhang**

### CODE REVIEW

**Comment by HaominZhang** [comment]
Also solved the issue metion in https://github.com/AppFolioOnboarding/Haomin_ImageUploader/pull/7#commitcomment-15694364 and https://github.com/AppFolioOnboarding/Haomin_ImageUploader/pull/7#commitcomment-15694372

File: app/models/picture.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Picture < ActiveRecord::Base
-  validates :link, :name, presence: true
+  validates :name, presence: true
   VALID_URL_PATTERN = /\A(http|https):\/\/[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(([0-9]{1,5})?\/.*)?\Z/ix
-  validates_format_of :link, with: VALID_URL_PATTERN
+  validates :link, presence: true, format: {with: VALID_URL_PATTERN}
```
Comment (rmacklin) [inline app/models/picture.rb]: :+1:

File: test/models/picture_test.rb [hunk 2]
```diff
@@ -29,7 +29,7 @@ def assert_name_cannot_be_blank(name)
   def assert_link_cannot_be_blank(link)
     picture = Picture.new(name: 'test', link: link)
     refute_predicate picture, :valid?
-    assert_equal(["can't be blank", "is invalid"], picture.errors.messages[:link])
+    assert_includes(picture.errors.messages[:link], "can't be blank")
```
Comment (rmacklin) [inline test/models/picture_test.rb]: :+1:

File: db/seeds.rb [hunk 3]
```diff
@@ -6,5 +6,23 @@
...
+                 {name: 'test11', link: 'https://media.giphy.com/media/9Em15XLdEkfSM/giphy.gif'},
+                 {name: 'test12', link: 'https://media.giphy.com/media/26tn8u4JaBPQmq8mY/giphy.gif'},
+                 {name: 'test13', link: 'https://media.giphy.com/media/q7lOaccQ68aly/giphy.gif'},
+                 {name: 'test14', link: 'https://media.giphy.com/media/3xz2BLBOt13X9AgjEA/giphy.gif'},
+                 {name: 'test15', link: 'https://media.giphy.com/media/Mc4PM6fMlwrTO/giphy.gif'},
+                 {name: 'test16', link: 'https://media.giphy.com/media/3o7asX38TrMOLWcVC8/giphy.gif'},
+                 {name: 'test17', link: 'https://media.giphy.com/media/PXvCWUnmqVdks/giphy.gif'},
+                 {name: 'test18', link: 'https://media.giphy.com/media/JEVqknUonZJWU/giphy.gif'},
+                 {name: 'test19', link: 'https://media.giphy.com/media/ocnViJ0lhu7EQ/giphy.gif'},
+                 {name: 'test20', link: 'https://media.giphy.com/media/Jr6CamswuuJdS/giphy.gif'}])
```
Comment (rmacklin) [inline db/seeds.rb]: Not a big deal but you could DRY this a bit by storing the URLs in a word array and calling `each` with a block to create the `Picture`s:

```ruby
image_urls = %w(
  https://media.giphy.com/media/kj5eW9ype6EV2/giphy.gif
  https://media.giphy.com/media/mNdOc0Aziv88E/giphy.gif
  https://media.giphy.com/media/3o85xk5IHPachXmpYQ/giphy.gif
  https://media.giphy.com/media/qPxSNx0LsSsZq/giphy.gif
  https://media.giphy.com/media/149EV8wlV75ZQc/giphy.gif
  https://media.giphy.com/media/INpdDYx0MH7RC/giphy.gif
  https://media.giphy.com/media/uZclwnfXqjpMk/giphy.gif
  https://media.giphy.com/media/NfUzifn8Z8yje/giphy.gif
  https://media.giphy.com/media/VT6eildjKdVWU/giphy.gif
  https://media.giphy.com/media/tgIhKRf5ELIje/giphy.gif
  https://media.giphy.com/media/9Em15XLdEkfSM/giphy.gif
  https://media.giphy.com/media/26tn8u4JaBPQmq8mY/giphy.gif
  https://media.giphy.com/media/q7lOaccQ68aly/giphy.gif
  https://media.giphy.com/media/3xz2BLBOt13X9AgjEA/giphy.gif
  https://media.giphy.com/media/Mc4PM6fMlwrTO/giphy.gif
  https://media.giphy.com/media/3o7asX38TrMOLWcVC8/giphy.gif
  https://media.giphy.com/media/PXvCWUnmqVdks/giphy.gif
  https://media.giphy.com/media/JEVqknUonZJWU/giphy.gif
  https://media.giphy.com/media/ocnViJ0lhu7EQ/giphy.gif
)

image_urls.each_with_index do |url, i|
  Picture.create!(name: "image#{i}", link: url)
end
```

**Pull request merged**